src/util/api.js

```diff
@@ -71,52 +72,29 @@ const API = {
     memo = '',
     amount = 0,
   }) {
-    return eosApi.transact({
-      actions: [{
-        account: 'eosio.token',
-        name: 'transfer',
-        authorization: [{
-          actor: from,
-          permission: 'active',
-        }],
-        data: {
-          from,
-          to,
-          quantity: EosPriceFormatter.formatPrice(amount),
-          memo,
-        },
-      }],
-    }, {
-      blocksBehind: 3,
-      expireSeconds: 30,
-    });
+    return eos().transfer(
+      currentEOSAccount().name,
```

Comment:
pass in from parameter is better, so that the tricky `currentEOSAccount()` is not needed.